test/sparse/sparse.jl

```diff
@@ -1642,9 +1642,9 @@ end
 
 # Test temporary fix for issue #16548 in PR #16979. Brittle. Expect to remove with `\` revisions.
 # This is broken by the introduction of RowVector... see brittle comment above.
-#@testset "issue #16548" begin
-#    @test which(\, (SparseMatrixCSC, AbstractVecOrMat)).module == Base.SparseArrays
-#end
+@testset "issue #16548" begin
+    @test_broken which(\, (SparseMatrixCSC, AbstractVecOrMat)).module == Base.SparseArrays
```

Comment:
Status of `\` revisions: Chances are no one has worked on improving the mechanism further since #16548 (particularly given that improving that mechanism depends on compiler improvements, see https://github.com/JuliaLang/julia/pull/16979#issuecomment-227935000).

Had a brief look at fixing this test. A simple fix would be to retrieve the method matches for `\` with `(SparseMatrixCSC, AbstractVecOrMat)` arguments and check that all such methods live in the `SparseArrays` module. For example, something along the lines of `all(meth -> meth.module == Base.SparseArrays, methods(\, (SparseMatrixCSC, AbstractVecOrMat)).ms)` could do the trick. Thoughts? Best!